distribution/docs/src/main/resources/content/_running/catalog-commands.adoc

```diff
@@ -0,0 +1,146 @@
+:title: Catalog Commands
+:type: subMaintaining
+:status: published
+:parent: Available System Console Commands
+:summary: Catalog commands available.
+:order: 00
+
+== {title}
+
+[WARNING]
+====
+Most commands can bypass the Catalog framework and interact directly with the Catalog provider if given the `--provider` option, if available.
+No pre/post plugins are executed and no message validation is performed if the `--provider` option is used.
+====
+
+.[[catalog_command_descriptions]]Catalog Command Descriptions
+[cols="1m,9a" options="header"]
+|===
+
+|Command
+|Description
+
+|catalog:describe
+|Provides a basic description of the Catalog implementation.
+
+|catlog:dump
+|Exports metacards from the local Catalog. Does not remove them. See <<_date_filtering_options,date filtering options>> below.
+
+|catalog:envlist
+|[IMPORTANT]
+====
+Deprecated as of ddf-catalog 2.5.0. Please use `platform:envlist`.
+====
+
+Provides a list of environment variables.
+
+|catalog:ingest
+|Ingests data files into the Catalog.
+
+|catalog:inspect
+|Provides the various fields of a metacard for inspection.
+
+|catalog:latest
+|Retrieves the latest records from the Catalog based on the Metacard.MODIFIED date.
```

Comment:
I think this should be `Core.MODIFIED` now to be in line with our normalized taxonomy

Reply:
fixed.